internal/distro/distro.go

```diff
@@ -141,11 +141,12 @@ func fromEnv(nameSuffix, defaultValue string) string {
 
 func bakedStringToBool(s string) bool {
 	// the linker only supports string args, so do some basic bool sensing
-	if s == "true" || s == "1" {
+	switch s {
```

Comment:
internal/distro/distro: Replace if/then/else with switch/case 868f0481f8dd84d9e1bc9a1da17b44445cbc307d

Could have lint details added to commit message, would be nice, though idk if necessary.